jenkins-builds/7.0.UP02-20.08-dom-gpu

```diff
@@ -21,6 +21,7 @@
  IDL-8.7.2-CSCS.eb
  Julia-1.5.0-CrayGNU-20.08-cuda.eb                      --set-default-module
  JuliaExtensions-1.5.0-CrayGNU-20.08-cuda.eb            --set-default-module
+ jupyterlab-1.2.16-CrayGNU-20.08-batchspawner-cuda.eb   --set-default-module
```

Comment:
```suggestion
 jupyterlab-1.2.16-CrayGNU-20.08-batchspawner-cuda.eb   --set-default-module
```
Thanks @rsarm: is `Jupyter 1.2.16 going to be the default? What about release `1.1.1`?